test/Microsoft.AspNet.Mvc.Core.Test/UrlHelperTest.cs

```diff
@@ -807,6 +809,83 @@ public void LinkWithCustomHostAndProtocol_ReturnsExpectedResult()
             Assert.Equal("https://myhost/named/home/newaction/someid", url);
         }
 
+        // Regression test for aspnet/Mvc#2859
```

Comment:
Haven't seen many (any?) links to closed bugs in our code. Is it about as useful as a TODO comment?